bin/sv_varcall_penalties.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+from pysam import VariantFile
+import cmdvcf
+import argparse
+from pprint import pprint
+from geneticmodels import GM
+
+MANTA_SIZE_FOR_PENALTY = 100000
+
+def main(args: object):
+    """
+    Read VCF line by line, add genetic models to INFO-field
+    """
+    vcf_object = VariantFile(args.input_vcf)
+    vcf_object.header.info.add(
+        "GQC",
+        ".",
+        "String",
+        "GATK genotype quality score when variant is only called by GATK"
+    )
+    vcf_object.header.info.add(
+        "MANTAPENALTY",
+        ".",
+        "String",
+        "big manta events not supported by gatk/cnvnator"
+    )
+    vcf_out = VariantFile(args.out_vcf, "w", header=vcf_object.header)
+    vcf_out.close()
+    with open(args.out_vcf, "a") as vcf_out:
+        for var in vcf_object.fetch():
+            var_dict = cmdvcf.parse_variant(var,vcf_object.header)
+            varcall_set = var_dict['INFO'].get('set',None)
+            if varcall_set is None:
+                exit("need a VCF merged with SVDB")
+            else:
+                varcallers_list = variant_called_by(varcall_set)
+                # GATK only caller, return average genotype qual for variant
+                if len(varcallers_list) == 1 and 'gatk' in varcallers_list:
+                    var_dict['INFO']['GQC'] = modify_gatk(var_dict)
+                # manta only caller, return bool if certain thingie
+                elif len(varcallers_list) == 1 and 'manta' in varcallers_list:
+                    mantapen = modify_manta(var_dict)
+                    if mantapen is not None:
+                        var_dict['INFO']['MANTAPENALTY'] = mantapen
+        
+                vcf_str = cmdvcf.vcf_string(var_dict,vcf_object.header)
+            vcf_out.write(vcf_str)
+
+def variant_called_by(varcallers: str):
```

Comment:
```suggestion
def variant_called_by(varcallers: str) -> List[str]:
```

`List` with import from `typing` library for up to Python 3.8.

`list` without import afterwards.